Makefile

```diff
@@ -165,13 +185,39 @@ push_images:
 		$(DOCKER) push $(REGISTRY)/$(TARGET):latest; \
 	fi
 
-push_manifest:
-	./manifest-tool push from-args --platforms $(PLATFORMS) --template $(REGISTRY)/$(TARGET)-ARCH:$(VERSION) --target $(REGISTRY)/$(TARGET):$(VERSION)
+gen_manifest:
+	mkdir build
+
+ifeq ($(PUSH_WINDOWS),true)
+	sed -e 's|TAG|$(IMAGE_VERSION)|g' \
+	-e 's|REGISTRY|$(REGISTRY)|g' \
+	-e 's/WIN_ONLY//g' \
+	manifest_spec.yaml.tmpl > ./build/manifest_spec.yaml;
+else
+	echo '$$PUSH_WINDOWS not set, not including Windows in manifest'
+	sed -e 's|TAG|$(IMAGE_VERSION)|g' \
+	-e 's|REGISTRY|$(REGISTRY)|g' \
+	-e '/^WIN_ONLY/d' \
+	manifest_spec.yaml.tmpl > ./build/manifest_spec.yaml;
+endif
+
+push_manifest: gen_manifest
+	./manifest-tool push from-spec ./build/manifest_spec.yaml
 
-push: pre containers
+push: #pre
```

Comment:
Debug; add back pre

Reply:
This should have a better solution; but `pre` does things that are CI specific (getting linux manifest-tool and logging in as sonobuoybot). When working locally we dont need that. Maybe we can just gate it instead, but still seems ugly.